zstacklib/zstacklib/utils/lvm.py

```
@@ -9,6 +9,7 @@
 import re
 import warnings
 import datetime
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)